src/librawspeed/common/Memory.h

```diff
@@ -69,6 +69,17 @@ alignedCalloc(size_t nmemb, size_t size) {
   return alignedMalloc<alignment>(size);
 }
 
+template <size_t alignment, typename T>
+// coverity[+alloc]
+inline void* __attribute__((malloc, warn_unused_result))
+alignedCalloc(size_t nmemb) {
```

Comment:
> Also, if your function already knows the type of the objects, I'd return a pointer to that type instead of void. Should make a bunch of c-casts unnecessary.

There are 9 calls of `alignedCalloc()`. Only in 4 cases, it makes sense.
I don't think it's worth it right now.